src/client/components/Competition.jsx

```diff
@@ -1,97 +1,92 @@
 import React, { useContext, useEffect, useState } from 'react';
 import { useNavigate } from 'react-router-dom';
-import { StoreContext } from '../utils/store';
 
 import { CardTag } from './card/CardTag';
 import { SeasonPodium } from './card/SeasonPodium';
 
-import { API_URL, HTTP_METHOD, LOCAL_STORAGE } from '../config';
+import { StoreContext } from '../utils/store';
 
 import './styling/competition.css';
 
-export const Competition = (props) => {
-    const { competition } = props;
+export const Competition = () => {
+    const { state } = useContext(StoreContext);
 
-    const { user } = useContext(StoreContext);
+    const { competitions, user } = state;
 
     const [currentSeason, setCurrentSeason] = useState([]);
     const [previousSeasons, setPreviousSeasons] = useState([]);
 
     const navigate = useNavigate();
 
-    console.log('states', { currentSeason, previousSeasons });
-
     useEffect(() => {
-        const fetchSeasons = async () => {
-            const token = localStorage.getItem(LOCAL_STORAGE.TOKEN);
-
-            const response = await fetch(API_URL.COMPETITION_GET_BY_ID, {
-                method: HTTP_METHOD.GET,
-                headers: {
-                    Authorization: token,
-                },
-            });
-
-            const result = await response.json();
+        const { seasons } = competitions[0];
 
-            console.log(result)
+        setCurrentSeason(seasons.slice(seasons.length - 1));
```

Comment:
What happens if the seasons array is empty? `seasons.slice(-1)` might have some interesting results